roles/elasticsearch/templates/elasticsearch.yml.j2

```
@@ -2,7 +2,13 @@ node.name: "{{ ansible_hostname }}"
 path.data: {{ elasticsearch_datapath }}
 path.logs: {{ elasticsearch_logpath }}
 network.host: ["_local_","_site_"]
-{% if elasticstack_release | int < 8 or groups['elasticsearch'] | length > 1 %}
+{% if ealsticsearch_http_publish_host is defined %}
+http.publish_host: "{{ ealsticsearch_http_publish_host }}"
+{% endif %}
+{% if ealsticsearch_http_publish_port is defined %}
+http.publish_port: {{ ealsticsearch_http_publish_port }}
+{% endif %}
+{% if elastic_release | int < 8 or groups['elasticsearch'] | length > 1 %}
```

```suggestion
{% if elasticstack_release | int < 8 or groups['elasticsearch'] | length > 1 %}
```

We had to rename this variable due to naming conventrion changes.